HashRegistrarSimplified.sol

```diff
@@ -444,6 +444,15 @@ contract Registrar {
     }  
 
     /**
+     * @dev Minimum length starts at 12 weeks reducing 1 letter every 4 weeks until min of 6
+     * @param timestamp The time for which you want to know the date.
+     */
+
+    function minLength(uint timestamp) constant returns (uint minLength) {
+        return uint minLength = max(6, 12 - (timestamp - registryStarted)/(4 weeks));
```

Comment:
A year after deployment (52 weeks), `12 - (timestamp - registryStarted) / (4 weeks)` will evaluate to a negative number, which will underflow, and the min domain length will become too long for practical use.